core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -311,6 +311,16 @@ private[spark] class DAGScheduler(
     eventProcessLoop.post(SpeculativeTaskSubmitted(task))
   }
 
+  /**
+   * Called by the TaskSetManager when there is an unschedulable blacklist task and dynamic
+   * allocation is enabled
+   */
+  def unschedulableBlacklistTaskSubmitted(
```

Comment:
I think we should rename.   Maybe just remove Submitted.  since this isn't really Task Submission

Reply:
I thought I removed everything, seems I missed this one. Thanks for pointing out.